packages/util-create-request/src/index.ts

```diff
@@ -0,0 +1,57 @@
+import {
+  CommandInput,
+  FinalizeHandler,
+  FinalizeHandlerArguments,
+  HttpRequest,
+  AWSClient,
+  Command,
+  MetadataBearer,
+  MiddlewareStack
+} from "@aws-sdk/types";
+
+export async function createRequest<
+  InputTypesUnion extends object,
+  InputType extends InputTypesUnion,
+  StreamType,
+  OutputType extends MetadataBearer = MetadataBearer
+>(
+  client: AWSClient<InputTypesUnion, MetadataBearer, StreamType>,
+  command: Command<
+    InputTypesUnion,
+    InputType,
+    MetadataBearer,
+    OutputType,
+    any,
+    StreamType
+  >
+): Promise<HttpRequest<StreamType>> {
+  const presignHandler: FinalizeHandler<
+    any,
+    HttpRequest<StreamType>,
+    StreamType
+  > = async (
+    args: FinalizeHandlerArguments<CommandInput>
+  ): Promise<HttpRequest<StreamType>> => {
+    return Promise.resolve(args.request);
```

Comment:
nit: The curly brackets can be removed as there's only one return statement

ditto on line 46, and in fixture.ts